src/anchorfm-pupeteer/index.js

```diff
@@ -9,19 +9,19 @@ function addUrlToDescription(youtubeVideoInfo) {
 
 async function setPublishDate(page, navigationPromise, date) {
   console.log('-- Setting publish date');
-  const publishDateButtonSelector = '//span[contains(text(),"Publish date:")]/following-sibling::button';
-  const [publishDateButton] = await page.$x(publishDateButtonSelector);
-  await publishDateButton.click();
+  await clickXpath(page,
+    '//span[contains(text(),"Publish date:")]/following-sibling::button',
+    env.UPLOAD_TIMEOUT)
```

Comment:
I don't think you need to supply the UPLOAD_TIMEOUT.

In fact, maybe don't supply timeout for these cases at all(though there is only one case where timeout was needed :D )

Reply:
I think timeout is not problem cause it only adds a period of time that the script will wait for the element. If element is already present timeout isn't used.